client/src/components/Input.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import SearchIcon from "../assets/icons/search.svg";
+import styled from "styled-components/macro";
+
+const InputContainer = styled.div`
+  width: 80%;
+  margin: 10px;
+  border: 1px solid #027368;
+  border-radius: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
```

Comment:
shouldnt it be a form, to use it with the inputfield in it?

```suggestion
const InputContainer = styled.form`
  width: 80%;
  margin: 10px;
  border: 1px solid #027368;
  border-radius: 50px;
  display: flex;
  align-items: center;
  justify-content: space-between;
`;
```

Reply:
Also true! Thanks again :)